CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 - Properly report [`newline-after-import`] when next line is a decorator
 - Fixed documentation for the default values for the [`order`] rule ([#601])
 
+### Added
+- [`exports-last`] lints that export statements are at the end of the file ([#620] + [#632])
```

Comment:
You'll need to rebase and move this back into the `Unreleased` section. Things have changed ever since you mad your PR :sweat_smile: 

If you don't know how to rebase, don't mind it, I'll do it later for you.